### PR TITLE
Don't double apply rotation

### PR DESCRIPTION
**Pull request by erikwijmans**
## Motivation and Context

Little bug in OBB loading for mp3d.  The radius of the obb is getting double corrected by the worldRotation.

## How Has This Been Tested

User has a visualization script, used that.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by erikwijmans** [comment]
> Maybe better to re-factor with a boolean (default true) to apply world rotation instead of undoing?

Yeah, that does seem cleaner.  Change made.

File: src/esp/scene/Mp3dSemanticScene.cpp [hunk 1]
```diff
@@ -94,29 +94,36 @@ bool SemanticScene::loadMp3dHouse(
...
+    if (applyRotation && hasWorldRotation) {
       p = rotation * p;
     }
     return p;
   };
 
   auto getBBox = [&](const std::vector<std::string>& tokens, int offset) {
-    return box3f(getVec3f(tokens, offset), getVec3f(tokens, offset + 3));
+    return box3f(getVec3f(tokens, offset),
+                 getVec3f(tokens, offset + 3).array().abs().matrix());
```
Comment (aclegg3) [inline src/esp/scene/Mp3dSemanticScene.cpp:111]: This seems unrelated to the rotation. Different bug?

Comment (erikwijmans) [reply src/esp/scene/Mp3dSemanticScene.cpp:111]: Oh yeah, just saw that we still hadn't fixed this -- after rotation, the sizes may be negative, which is weird -- so just did it.
